### PR TITLE
Update user PATCH route

### PR DESCRIPTION
**Pull request by andra5980**

### CODE REVIEW

File: controllers/userController.js [hunk 1]
```diff
@@ -62,6 +63,21 @@ exports.getUsers = async (req, res) => {
 
 exports.updateUser = async (req, res) => {
   try {
+    const idToSearchFor = mongoose.Types.ObjectId(req.user['user:id'])
```
Comment (procateodor) [inline controllers/userController.js]: We have a constant for that in `constants` folder

**Review by procateodor — APPROVED** [review]
One small comment.

**Review by procateodor — CHANGES_REQUESTED** [review]
Kind reminder.

File: controllers/userController.js [hunk 2]
```diff
@@ -62,6 +63,21 @@ exports.getUsers = async (req, res) => {
...
+        success: false,
+        message: 'User not found!'
+      })
+    }
+
+    await req.db.User.updateOne(user, req.body)
+
+    return res.json({
+      success: true
+    })
```
Comment (adriangotca98) [inline controllers/userController.js]: Add a message with this please

File: controllers/userController.js [hunk 3]
```diff
@@ -62,6 +63,21 @@ exports.getUsers = async (req, res) => {
...
   } catch (error) {
     req.log.error(`Unable to update user -> ${error}`)
     return res.status(HttpStatus.INTERNAL_SERVER_ERROR).json({
```
Comment (adriangotca98) [inline controllers/userController.js:90]: Here on the 500 response, also a message

**Review by adriangotca98 — CHANGES_REQUESTED** [review]

**Comment by procateodor** [comment]
[Update user data route](https://trello.com/c/vhTsCDYb/18-update-user-data-route)

**Review by procateodor — DISMISSED** [review]
LGTM

File: controllers/userController.js (hunk 3 above)
Comment (adriangotca98) [reply controllers/userController.js:90]: on the 500 response, there should be a message. It wasn't visible at the time of my review, please add a message to the response. Sorry for misunderstanding this

**Review by adriangotca98 — CHANGES_REQUESTED** [review]

**Review by adriangotca98 — APPROVED** [review]

**Pull request merged**